website/docs/docs/build/metricflow-commands.md

```diff
@@ -60,6 +60,7 @@ You can install [MetricFlow](https://github.com/dbt-labs/metricflow#getting-star
 
 </Tabs>
 
+Something to note, MetricFlow `mf` commands return an error if you have a Metafont latex package installed. To run `mf` commands, uninstall the package.
```

Comment:
hey @Jstein77 , does this also apply to dbt cloud cli `dbt sl` commands too?

Reply:
Yea, i think the dependencies are the same.